src/terraform/install.sh

```diff
@@ -180,23 +180,29 @@ check_packages() {
 # Function to fetch the version released prior to the latest version
 get_previous_version() {
     REPO_URL=$1
-    curl -s "${REPO_URL}/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/'
+    if command -v jq &> /dev/null; then
```

Comment:
We can always install `jq`, I'd recommend installing jq with check-packages and removing this conditional block.

Reply:
done